BusinessCentral.LinterCop/Design/Rule0077MissingParenthesis.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.Dynamics.Nav.CodeAnalysis;
+using Microsoft.Dynamics.Nav.CodeAnalysis.Diagnostics;
+using Microsoft.Dynamics.Nav.CodeAnalysis.Syntax;
+using System.Collections.Immutable;
+using BusinessCentral.LinterCop.AnalysisContextExtension;
+
+namespace BusinessCentral.LinterCop.Design;
+
+[DiagnosticAnalyzer]
+public class Rule0077MissingParenthesis : DiagnosticAnalyzer
+{
+    private static readonly ImmutableHashSet<string> MethodsRequiringParenthesis = ImmutableHashSet.Create(
```

Comment:
I'm comparing the Method name, not the identifier itself, so it will always be in the "correct" case. 
But if you'd like, I can add a testcase with different casings?

Reply:
You're right! I didn't realize it's the Identifier itself which is static. No need to handle casing.